<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>499</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>688</width>
    <height>661</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A2M2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
	QLabel {
		margin: 0px;
		padding: 5px;
		border: 0px;
	}

	QLineEdit{ 
		margin: 0px;
		padding: 5px;
	}</string>
   </property>
   <widget class="QTabWidget" name="Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="SessionTab">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Sesión de Captura</string>
     </attribute>
     <widget class="QPushButton" name="createSessionButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0079bf; 
color:white;</string>
      </property>
      <property name="text">
       <string>Crear Sesión</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Datos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>309</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startSessionButton">
         <property name="styleSheet">
          <string notr="true">background-color: #0079bf; 
color:white;</string>
         </property>
         <property name="text">
          <string>Empezar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelSessionButton">
         <property name="styleSheet">
          <string notr="true">background-color: #0079bf; 
color:white;</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uploadSessionButton">
         <property name="styleSheet">
          <string notr="true">background-color: #0079bf; 
color:white;</string>
         </property>
         <property name="text">
          <string>Subir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addFieldButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0079bf; 
color:white;</string>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="configButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0079bf; 
color:white;</string>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="pathButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0079bf; 
color:white;</string>
      </property>
      <property name="text">
       <string>Ruta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Parámetros</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Directorio</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CameraTab">
     <attribute name="title">
      <string>Cámara Thor</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>301</width>
        <height>163</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="cameraContainer">
       <item>
        <layout class="QHBoxLayout" name="velocityContainer_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="directoryLabel">
             <property name="text">
              <string>Directorio</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="cameraThorDirectory">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	margin: 5px 0px 5px 0px; 
}</string>
             </property>
             <property name="text">
              <string>/home/</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="cameraThorNameFiles">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	margin: 5px 0px 5px 0px; 
}</string>
             </property>
             <property name="text">
              <string>imagen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nameFilesLabel">
             <property name="text">
              <string>Nombre archivos</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="cameraThorNPhotos"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="nPhotosLabel">
             <property name="text">
              <string>Número fotos</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonsContainer_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="cameraCaptureButton">
           <property name="styleSheet">
            <string notr="true">background-color: #0079bf; 
color:white;</string>
           </property>
           <property name="text">
            <string>Capturar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="SliderTab">
     <attribute name="title">
      <string>Slider</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>341</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="sliderContainer">
       <item>
        <layout class="QHBoxLayout" name="directionContainer">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="directionLabel">
             <property name="text">
              <string>Dirección</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="distanceValue">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	margin: 5px 0px 5px 0px; 
}</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="distanceLabel">
             <property name="text">
              <string>Distancia</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="velocityLabel">
             <property name="text">
              <string>Velocidad</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="directionValue"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="velocityValue">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	margin: 5px 0px 5px 0px; 
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonsContainer">
         <item>
          <widget class="QPushButton" name="sliderMoveButton">
           <property name="styleSheet">
            <string notr="true">background-color: #0079bf; 
color:white;</string>
           </property>
           <property name="text">
            <string>Mover</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sliderStopButton">
           <property name="styleSheet">
            <string notr="true">background-color: #0079bf; 
color:white;</string>
           </property>
           <property name="text">
            <string>Detener</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="sliderHomeButton">
         <property name="styleSheet">
          <string notr="true">background-color: #0079bf; 
color:white;</string>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
